inst/www/shared/shiny.js

```diff
@@ -3477,7 +3466,6 @@ $(document).on('click', '.slider-animate-button', function(evt) {
           val.to = Math.min(slider.result.max, slider.result.to + slider.options.step);
 
         slider.update(val);
-        forceIonSliderUpdate(slider);
```

Comment:
I see. I think we can just run grunt on Travis and automatically push to the repo. That way, we no longer have to worry about these minified js files and map files. It is pretty easy to set up, but I guess you'd prefer running grunt by ourselves locally.